google-cloud-functions-http-quickstart/src/main/resources/META-INF/resources/index.html

```diff
@@ -135,7 +135,7 @@ <h3>Application</h3>
             <ul>
                 <li>GroupId: org.acme</li>
                 <li>ArtifactId: google-cloud-functions-http</li>
-                <li>Version: 1.0-SNAPSHOT</li>
+                <li>Version: 1.0.0-SNAPSHOT</li>
                 <li>Quarkus Version: 1.6.1.Final</li>
```

Comment:
@gsmet is that normal `1.6.1.Final`?

Reply:
I think they are updated when pushed to the master branch.